includes/class-wp-job-manager-post-types.php

```diff
@@ -736,19 +738,45 @@ public static function get_permalink_structure() {
 			switch_to_locale( get_locale() );
 		}
 
-		$permalinks = wp_parse_args(
-			(array) get_option( 'wpjm_permalinks', array() ),
+		/**
+		 * Option `wpjm_permalinks` was renamed to match other options in 1.32.0.
+		 *
+		 * Reference to the old option and support for non-standard plugin updates will be removed in 1.34.0.
+		 */
+		$legacy_permalink_settings = '[]';
```

Comment:
I wonder if it would make sense to consolidate this code with the similar code in `includes/admin/class-wp-job-manager-permalink-settings.php`? Maybe we call a function that checks for `wpjm_permalinks`, converts it to the `job_manager_permalinks` option if needed, and deletes the old option. Then we just call it from here and from `includes/admin/class-wp-job-manager-permalink-settings.php` to ensure that we're dealing with the up-to-date option.

Not a blocker for me, but it would reduce a bit of code duplication.

Reply:
done in 0df7616477c624018edfb32bf189c1feb0481c15